spec/models/user_spec.rb

```diff
@@ -3,4 +3,86 @@
 RSpec.describe User do
   it { should validate_presence_of :email }
   it { should validate_presence_of :password }
+  it { should have_many :questions }
+  it { should have_many :answers }
+  it { should have_many :authorizations }
+
+  describe '.find_for_oauth' do
+    let!(:user) { create(:user) }
+    let(:auth) { OmniAuth::AuthHash.new(provider: 'facebook', uid: '123456') }
+
+  	context 'user already has authorization' do
+  		it 'returns the user' do
+  			user.authorizations.create(provider: 'facebook', uid: '123456')
+  			expect(User.find_for_oauth(auth)).to eq user
+      end
+    end
```

Comment:
`end` at 19, 4 is not aligned with `context 'user already has authorization' do` at 14, 3